__tests__/api/costumes.test.js

```diff
@@ -40,4 +41,28 @@ describe('GET api/costumes', () => {
         expect(response.body.costumes[1].name).toBe('buttless chaps');
         expect(response.body.costumes[2].name).toBe('bonnet');
       })
-})
\ No newline at end of file
+})
+
+
+describe('GET api/costumes/:id', () => {
+    it('returns costume by ID', async () => {
+        await createTables(pool);
```

Comment:
Put this in a beforeEach if possible (maybe already done in a later PR?)

Reply:
This may have happened already in another PR. I'll be sure that happens in main.